descriptions/rep/en_us.lua

```diff
@@ -1,424 +1,421 @@
 	[570] = {"570", "Playdough Cookie", "Each of Isaac's tears have a different color and status effect"}, -- Playdough Cookie
-	[571] = {"571", "Orphan Socks", "Grants immunity to creep and floor spikes#↑ {{Speed}} +0.3 Speed up#↑ {{SoulHeart}} +2 Soul Hearts"}, -- Orphan Socks
-	[572] = {"572", "Eye of the Occult", "Isaac's tears can be controlled in mid-air#↑ {{Damage}} +1 Damage up#↑ {{Range}} +2 Range up#↓ {{Shotspeed}} -0.16 Shot Speed down"}, -- Eye of the Occult
-	[573] = {"573", "Immaculate Heart", "↑ {{Heart}} +1 Health up#{{Heart}} Full health#↑ {{Damage}} +20% Damage up#20% chance to shoot an additional orbiting spectral tear"}, -- Immaculate Heart
-	[574] = {"574", "Monstrance", "Isaac is surrounded by two damaging auras#The inner aura deals more damage"}, -- Monstrance
-	[575] = {"575", "The Intruder", "Buries itself in Isaac's head and shoots 4 extra slowing tears#Taking damage can make the spider exit the head and chase enemies"}, -- The Intruder
+	[571] = {"571", "Orphan Socks", "↑ {{Speed}} +0.3 Speed#↑ {{SoulHeart}} +2 Soul Hearts#Immune to creep and floor spikes"}, -- Orphan Socks
+	[572] = {"572", "Eye of the Occult", "↑ {{Damage}} +1 Damage#↑ {{Range}} +2 Range#↓ {{Shotspeed}} -0.16 Shot speed#Isaac's tears can be controlled in mid-air"}, -- Eye of the Occult
+	[573] = {"573", "Immaculate Heart", "↑ {{Heart}} +1 Health#↑ {{Damage}} x1.2 Damage multiplier#{{Heart}} Full health#20% chance to shoot an extra orbiting spectral tear"}, -- Immaculate Heart
+	[574] = {"574", "Monstrance", "Isaac is surrounded by a damaging aura#The closer enemies are to Isaac, the more damage the aura deals to them"}, -- Monstrance
+	[575] = {"575", "The Intruder", "{{Slow}} Buries itself in Isaac's head and shoots 4 extra slowing tears#Taking damage can make the spider exit the head and chase enemies"}, -- The Intruder
 	[576] = {"576", "Dirty Mind", "All Dip (small poop) enemies are friendly#Destroying poop spawns 1-4 Dips#Dip type depends on the poop type#Rocks may be replaced with poop"}, -- Dirty Mind
-	[577] = {"577", "Damocles", "{{Warning}} SINGLE USE {{Warning}}#Hangs a sword above Isaac's head, which doubles all pedestal items#Does not double Shop, Chest, or Devil deal items#{{Warning}} After taking any damage, the sword has a chance to instantly kill Isaac every frame"}, -- Damocles
-	[578] = {"578", "Free Lemonade", "Creates a large pool of yellow creep"}, -- Free Lemonade
-	[579] = {"579", "Spirit Sword", "Instead of shooting tears, Isaac swings a sword#The sword deals 3x damage#{{Chargeable}} Charging does a spin attack + projectile shot#Fire projectiles with normal swings at full health"}, -- Spirit Sword
+	[577] = {"577", "Damocles", "!!! SINGLE USE !!!#Hangs a sword above Isaac's head, which doubles all pedestal items#Does not double shop, chest, or devil deal items#!!! After taking any damage, the sword has an extremely low chance to instantly kill Isaac every 0.133 seconds"}, -- Damocles
```

Comment:
0.133 seconds is way to specific and not really "graspable" for every player. Frame is more understandable i think